examples/spmd/run_spmd.py

```diff
@@ -143,7 +143,7 @@ def work_main(rank: int, world_size: int) -> None:
                 ),
                 placements=[Replicate()],
             ),
-            optimize_first_iter=True,
+            force_compile=True,
```

Comment:
I wanted to check that this change was intentional (since I am not sure if `run_spmd.py` runs in CI).

It looks like the `SPMD` API uses `optimize_first_iter`, not `force_compile`.

https://github.com/pytorch/tau/blob/7c71c23e6aea3f69f98859921447b6eea74285b2/spmd/compiler/api.py#L20-L28

Reply:
I had moved to rename this param but reverted as you can see. thanks for catching it!